plugin/cassandra/plugin.go

```diff
@@ -90,106 +90,151 @@ package main
 import (
 	"os"
 	"path/filepath"
+	"sort"
 	"strings"
 
-	fmt "github.com/jhunt/go-ansi"
+	fmt "github.com/jhunt/go-ansi" 
 
 	"github.com/starkandwayne/shield/plugin"
 )
 
+// Default configuration values for the plugin
 const (
-	DefaultHost     = "127.0.0.1"
-	DefaultPort     = "9042"
-	DefaultUser     = "cassandra"
-	DefaultPassword = "cassandra"
-	DefaultDataDir  = "/var/vcap/store/cassandra/data"
-
+	DefaultHost      = "127.0.0.1"
+	DefaultPort      = "9042"
+	DefaultUser      = "cassandra"
+	DefaultPassword  = "cassandra"
+	DefaultSaveUsers = true
+	DefaultDataDir   = "/var/vcap/store/cassandra/deployment_name/data"
+	DefaultBinDir    = "/var/vcap/jobs/cassandra/bin"
 	VcapOwnership = "vcap:vcap"
 	SnapshotName  = "shield-backup"
 )
 
+// Array or slices aren't immutable by nature; you can't make them constant
+var (
+	DefaultExcludeKeyspaces = []string{"system_schema", "system_distributed", "system_auth", "system", "system_traces"}
+	SystemAuthTables        = []string{"roles", "role_permissions", "role_members", "resource_role_permissons_index"}
+)
+
 func main() {
 	p := CassandraPlugin{
 		Name:    "Cassandra Backup Plugin",
 		Author:  "Orange",
-		Version: "0.1.0",
+		Version: "1.0.0",
```

Comment:
Version bumps are not necessary; they are not actually communicated anywhere.  Plugin versions are locked to the version of SHIELD that is being distributed.